accessibility_api/accessibility_lib/scripts/debug.py

```diff
@@ -0,0 +1,42 @@
+'''
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this file,
+You can obtain one at http://mozilla.org/MPL/2.0/.
+'''
+
+from .constants import CHILDID_SELF
+
+DEBUG_ENABLED = False
```

Comment:
Since this is where we set the flag, lets have the ```if DEBUG_ENABLED``` check that you have in other files inside each print function. This way you can add prints without the check where you want, and they will only run if it's set to true.